systemtest/src/main/java/io/strimzi/systemtest/kafkaclients/externalClients/KafkaClientProperties.java

```diff
@@ -292,4 +505,69 @@ private static File createKeystore(byte[] ca, byte[] cert, byte[] key, String pa
         keystore.deleteOnExit();
         return keystore;
     }
+
+    /**
+     * Use Keycloak Admin API to update Authorization Services 'decisionStrategy' on 'kafka' client to AFFIRMATIVE
+     *
+     * @throws IOException
+     */
+    static void fixBadlyImportedAuthzSettings() throws IOException {
```

Comment:
If this is to work around a bug in keycloak you should include include a link to the bug report in keycloak (and ideally watch that bug so we know when it gets fixed and this workaround can be removed)